CMakeLists.txt

```diff
@@ -10,9 +10,9 @@ file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include)
 
 set(VERSION 0.3.4)
 ExternalProject_Add(glog_src
-  DOWNLOAD_COMMAND wget --retry-connrefused --waitretry=1 --timeout=12 --tries 10 https://github.com/google/glog/archive/v${VERSION}.zip
```

Comment:
@helenol are you sure the DOWNLOAD_COMMAND was the source of the rebuild and not the UPDATE_COMMAND?